docs/v0.14/performance-tuning.txt

```diff
@@ -0,0 +1,71 @@
+# Performance Tuning
+
+## Check top command
+
+If Fluentd doesn't perform as well as you had expected, please check the `top` command first. You need to identify which part of your system is the bottleneck (CPU? Memory? Disk I/O? etc).
+
+## Avoid extra computations
+
+This is more like a general recommendation, but it’s always better **NOT TO HAVE** extra computation inside Fluentd. Fluentd is flexible to do quite a bit internally, but adding too much logic to Fluentd’s configuration file makes it difficult to read and maintain, while making it also less robust. The configuration file should be as simple as possible.
+
+## Use num_threads parameter
+
+If the destination for your logs is a remote storage or service, adding a `num_threads` option will parallelize your outputs (the default is 1). Using multiple threads can hide the IO/network latency. This parameter is available for all output plugins.
+
+    :::text
+    <match test>
+      @type output_plugin
+      num_threads 8
```

Comment:
Use `<buffer>` and `flush_thread_count`.

Reply:
I've fixed them.
But `<buffer ...>` is preferable for me instead of `<buffer>`.
Because `<buffer>` directive's attributes is often used in v0.14 ready output plugins.